### ISSUE TITLE
Header formatting not consistent with doc

### ISSUE THREAD

**Issue by droptablestar** [aug-riedinger/google-docs-publisher#28]
I have a header which I have removed spacing before and after the paragraphs inside and the spacing in the generated doc doesn't seem to reflect the changes. The header is still spaced out as if the spaces weren't removed.

**Comment by augnustin** [comment]
Hi again,

Are you aware that it sometimes takes up to 5-10 minutes for your document to update?

Again, if you want me to investigate, I'll need an example + making sure it is in the non-formatted published version.

Best

**Comment by droptablestar** [comment]
Yes, indeed. I've waited :)

Additionally, I created a new doc to test the horizontal line problem from #27 so I could add a header to a fresh doc. No header but the horizontal lines are there.

**Comment by augnustin** [comment]
Regarding this issue, I'm not sure I understood what goes wrong. 

What do you call a header? What doesn't appear?

Best

**Comment by droptablestar** [comment]
Right, this got confusing. I have two docs and I'll link the original Google Doc, the doc published using the vanilla Google Doc publish, and the doc published with google-doc-publisher for each. 

What I'm noticing: 

- The header for both of these docs is missing from both the regular published version and the gdocs version.
- On the resume version the horizontal lines don't appear on both the regular published version and the gdocs published version but they do appear on the horizontal line document.

What I've tried:

- Removing custom line spacing from header.
- Republishing.
- Creating new docs.

**Horizontal line test**
[Google Doc](https://docs.google.com/document/d/11rQZK67RVxQEuNNeFCMjsPQmuNXD0GSh7DY7Zmoo_rA/edit)
[Google Doc Published](https://docs.google.com/document/d/e/2PACX-1vQQ3XpgHI4oxvqU9q33NjYpi3O5e-xU8a4s8_5NaGOuA4r7QIYBKPGxGjTmbez0ZkJSfaTsJ_FlAOi8/pub)
[gdoc](https://docs.google.com/document/d/e/2PACX-1vQQ3XpgHI4oxvqU9q33NjYpi3O5e-xU8a4s8_5NaGOuA4r7QIYBKPGxGjTmbez0ZkJSfaTsJ_FlAOi8/pub)

**Resume** 
[Google Doc](https://docs.google.com/document/d/1DMOcYOThTL1FtMJVnmf71xgIyxsYQkzW-7srIa-pAx0/edit#heading=h.s34liub7k1sn)
[Google Docs Published](https://docs.google.com/document/d/e/2PACX-1vRPnbgxLinIFEBEp6W-0lULgQFBQiM_RuV2ttBewyEBNcjLOua7VtrSvITAoaiFFsuhNyy-b7CF_8ia/pub)
[gdoc](https://gdoc.pub/doc/1DMOcYOThTL1FtMJVnmf71xgIyxsYQkzW-7srIa-pAx0)
